x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/cli/CliIntegrationTestCase.java

```diff
@@ -59,6 +60,7 @@ protected SecurityConfig securityConfig() {
     protected void index(String index, CheckedConsumer<XContentBuilder, IOException> body) throws IOException {
         Request request = new Request("PUT", "/" + index + "/doc/1");
```

Comment:
I'm wondering if we could we switch this to `_doc` instead of expecting a warning?

Reply:
@jtibshirani  Thanks Julie. Changing this into a `_doc` in this file or in `JdbcIntegrationTestCase.java` now will lead to failures of many sql tests. I am planning to have a subsequent PR that can go through our xpack products and change them to use non-deprecated versions.

Reply:
Making these changes in a follow-up PR works for me. I'll add a note to the meta issue to track this. It could be that the individual teams should make these changes, as I think it may require more than just updating integration tests. For example, the security indices are still created using the old type name `doc`.